activerecord/lib/active_record/connection_adapters/sqlite3/quoting.rb

```diff
@@ -12,10 +12,6 @@ def quote_table_name_for_assignment(table, attr)
           quote_column_name(attr)
         end
 
-        def quote_table_name(name)
```

Comment:
This removes memoization, which I think we don't want. Please revert this line and only remove the `gsub`.

Reply:
I undo things to exactly how things were before your fix. `supers just ends up calling `quote_column_name`, which does cache things. I don't mind changing it back if you prefer. Let me know.

Reply:
Yes, [abstract quoting](https://github.com/rails/rails/blob/dfa857ef25b5abf7485e4f7115e4a311e00c599c/activerecord/lib/active_record/connection_adapters/abstract/quoting.rb#L72) calls `quote_column_name ` but it uses the wrong ivar. I would prefer keeping them separate as they were before.

Reply:
Okay, but then do I use `super` or not? Because if I do, then both ivar will cache the value which feels wasteful.